router/pkg/otel/attributes.go

```diff
@@ -46,10 +47,11 @@ const (
 	WgIsBatchingOperation              = attribute.Key("wg.operation.batching.is_batched")
 	WgBatchingOperationsCount          = attribute.Key("wg.operation.batching.operations_count")
 	WgBatchingOperationIndex           = attribute.Key("wg.operation.batching.operation_index")
-
 	// HTTPRequestUploadFileCount is the number of files uploaded in a request (Not specified in the OpenTelemetry specification)
 	HTTPRequestUploadFileCount = attribute.Key("http.request.upload.file_count")
 
+	WgConnReused = attribute.Key("wg.conn.reused")
```

Comment:
We should avoid shortcuts in attribute names. Proposal `wg.http.client.reused_connection`

Similar metric: https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpclientopen_connections